### ISSUE TITLE
docs: update for v2

### ISSUE THREAD

**Issue by kamranayub** [senacor/azure-function-middleware#340]
Hi, thank you for making this great library! I noticed after upgrading to 2.x today that the signatures changed but the README isn't updated yet, so I was a bit confused until I looked at the `examples` directory 👍 

### `MiddlewareFunction` is no longer an export

Instead, you can use `AzureFunction` from the `@azure/functions` package.

```diff
- import { MiddlewareFunction } from "@senacor/azure-function-middleware"
+ import { AzureFunction } from "@azure/functions"
```

### `middleware` function changed to have before/after handlers

```diff
- export default middleware(timerTrigger, [handler]);
+ export default middleware([handler /* before */], timerTrigger, [/* after */]);
```

**Comment by maaaNu** [comment]
Hi  Kamran Ayub,

Thank you for this issue! I will improve the README.md till friday and create Pull-Request.

**Comment by maaaNu** [comment]
Fixed in be1cf80df5b82759f9b83d53d0faed2ea19ed31d
